src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -969,13 +971,15 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <_ProjectDefaultTargets Condition="'$(MSBuildProjectDefaultTargets)' == ''">Build</_ProjectDefaultTargets>
 
     <RebuildDependsOn>
+      $(RebuildDependsOn);
```

Comment:
I wonder about this one. Rebuild has such drastic effects, and the order of targets is absolutely critical. If a user were to want to add targets to Rebuild, whether their target runs as part of the clean phase or the subsequent build phase makes a *huge* difference. IMO positioning `$(RebuildDependsOn)` at the front of the list is arbitrary and as likely to be wrong as right, and therefore shouldn't be added at all. Let users inject their target as a dependent target to one of the more specific targets.